databricks/koalas/missing/indexes.py

```diff
@@ -194,6 +148,52 @@ class _MissingPandasLikeMultiIndex(object):
     get_values = unsupported_function('get_values', deprecated=True)
     contains = unsupported_function('contains', deprecated=True)
     item = unsupported_function('item', deprecated=True)
+    append = unsupported_function('append', deprecated=True)
```

Comment:
@RainFung, seems it's not deprecated. It's just not documented: https://github.com/pandas-dev/pandas/blob/master/pandas/core/indexes/multi.py#L1997-L2008